docs/guide/your-first-scene.md

```diff
@@ -10,13 +10,13 @@ This guide will help you to create your first Tres scene. 🍩
 
 ## Setting up the experience Canvas
 
-Before we can create a Scene, we need somewhere to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `canvas` HTML element to mount the `WebglRenderer` and initialize the `scene`
+Before we can create a scene, we need a space to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `canvas` HTML element to mount the `WebglRenderer` and initialize the `scene`
 
 With **TresJS** you only need to import the default component `<TresCanvas />` and add it to the template of your Vue component.
 
 ```vue
 <script lang="ts" setup>
-import { TresCanvas } from '@tresjs/core'
+import { TresCanvas } from "@tresjs/core";
```

Comment:
It'd be best to use the project's linter rules here, I think.

The linter rules (node_modules/@tresjs/eslint-config-base/index.js) are set up to use single quotes and omit optional semicolons.

Same for the other corrected code examples. (I won't add comments as it seems passive-aggressive, lol. 😀 Though if it would be helpful, I can do that.)

Reply:
Thanks, yes absolutely - I think I missed those before I disabled my save actions in VsCode - might be helpful to also export the config settings for the "action on save" in the future. Or disable that "format on save" feature if possible.